### PR TITLE
WRO-12546: Update storybook6.5 and webpack5 to agate sampler.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [X] Documentation was verified or is not changed
* [X] UI test was passed or is not needed
* [X] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated storybook version to the latest and builder to webpack5 to agate sampler. I
tested and fixed all stories for any warning/error. I tested global controls change.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-12546

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Comment by MikyungKim** [comment]
@enact/storybook-utils 5.0.2 is released with `storyStoreV7` option bug. Would you please remove `storyStoreV7` option and update the @enact/storybook-utils version to 5.0.2 and storybook modules to ^6.5.14?

**Comment by daniel-stoian-lgp** [comment]
> @enact/storybook-utils 5.0.2 is released with `storyStoreV7` option bug. Would you please remove `storyStoreV7` option and update the @enact/storybook-utils version to 5.0.2 and storybook modules to ^6.5.14?

sure, i was just doing that as well :)

File: samples/sampler/.qa-storybook/preview.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ export const parameters = {
 		container: DocsContainer,
 		page: DocsPage,
 		iframeHeight: 360,
+		inlineStories: false,
```
Comment (MikyungKim) [inline samples/sampler/.qa-storybook/preview.js:50]: Should we do the same as https://github.com/enactjs/sandstone/blob/develop/samples/sampler/.storybook/preview.js#L57 for agate too?

Comment (daniel-stoian-lgp) [reply samples/sampler/.qa-storybook/preview.js:50]: we have WRP-4945 which is just waiting for this PR to be merged :)

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**